apps/spotlight/src/styles.css

```diff
@@ -1 +1,69 @@
 @import '@atj/design/static/uswds/styles/styles.css';
+
+/* Splash Page */
+
+.offline-homepage .grid-container {
+    max-width: 64rem;
+}
+
+.offline-homepage .usa-hero {
+    background-image: url("../src/images/video-screenshot-gray-80.jpg");
+}
+
+.usa-modal--lg {
```

Comment:
Rather than extending the `usa-modal` styles globally, could we define our own class for these styles?

Reply:
Done. I added the page class name to that rule so that it only affects the modal in the splash page. Thanks for catching that.